circle.yml

```diff
@@ -115,8 +121,6 @@ test:
   # Aggregate and report to coveralls
     - gvm use stable; go list ./... | xargs -L 1 -I{} cat "$GOPATH/src/{}/coverage.out" | grep -v "$CIRCLE_PAIN" >> ~/goverage.report:
```

Comment:
Shouldn't we specify the build tags here as well?
Admittedly we don't do anything with the coverage report but then...